src/test/java/edu/harvard/iq/dataverse/EMailValidatorTest.java

```diff
@@ -37,6 +37,11 @@ public void testIsEmailValid() {
         assertEquals(false, EMailValidator.isEmailValid("dora@.com", null));
         assertEquals(false, EMailValidator.isEmailValid("", null));
         assertEquals(false, EMailValidator.isEmailValid(null, null));
+        /*
+        Add tests for 4601
```

Comment:
We should reformat this emails in the test to be a little less "real".

Reply:
I added a couple that came directly from the RT tickets referenced in the issue (that are now passing validation). What else did you have in mind?

Reply:
@mheppler kudos on catching that. Fixed in 0c52d26 it looks like.

Reply:
👍